tests/unit/EntityHeader.spec.ts

```diff
@@ -273,6 +273,172 @@ describe('Entity Header - data', () => {
+    businessStore.setLegalName(null)
+    rootStore.entityStatus = EntityStatus.DRAFT_AMALGAMATION
+    businessStore.setLegalType(CorpTypeCd.BC_COMPANY)
+    filingHistoryListStore.setFilings([])
+    rootStore.setTasks([
+      {
+        task: {
+          filing: {
+            displayName: 'BC Limited Company Amalgamation Application - Horizontal'
+          }
+        }
+      } as any
+    ])
+
+    const wrapper = shallowMount(EntityHeader, {
+      vuetify,
+      router,
+      propsData: { businessId: null, tempRegNumber: 'T1234567' }
+    })
+    await Vue.nextTick()
+
+    expect(wrapper.find('#app-name').text()).toBe('Numbered Amalgamated Company')
+    expect(wrapper.find('#app-description').text()).toBe('BC Limited Company Amalgamation Application - Horizontal')
+  })
+
+  it('displays filed numbered amalgamated company - Horizontal', async () => {
+    // set store properties
+    businessStore.setLegalName(null)
+    rootStore.entityStatus = EntityStatus.FILED_AMALGAMATION
+    businessStore.setLegalType(CorpTypeCd.BC_COMPANY)
+    rootStore.setTasks([])
+    filingHistoryListStore.setFilings([
+      {
+        displayLedger: true,
+        displayName: 'BC Limited Company Amalgamation Application - Horizontal',
+        effectiveDate: '2019-06-02 19:22:59 GMT',
+        name: FilingTypes.AMALGAMATION_APPLICATION,
+        status: FilingStatus.COMPLETED,
+        submittedDate: 'Sun, 02 Jun 2019 19:22:59 GMT'
+      } as any
+    ])
+
+    const wrapper = shallowMount(EntityHeader, {
+      vuetify,
+      router,
+      propsData: { businessId: null, tempRegNumber: 'T1234567' }
+    })
+    await Vue.nextTick()
+
+    expect(wrapper.find('#app-name').text()).toBe('Numbered Amalgamated Company')
+    expect(wrapper.find('#app-description').text()).toBe('BC Limited Company Amalgamation Application - Horizontal')
+  })
+
+  it('displays numbered amalgamated company - Horizontal amalgamation', async () => {
+    // set store properties
+    businessStore.setLegalName(null)
+    rootStore.entityStatus = null
+    businessStore.setLegalType(CorpTypeCd.BC_COMPANY)
+    rootStore.setTasks([])
+    filingHistoryListStore.setFilings([
+      {
+        displayLedger: true,
+        displayName: 'BC Limited Company Amalgamation Application - Horizontal',
+        effectiveDate: '2019-06-02 19:22:59 GMT',
+        name: FilingTypes.AMALGAMATION_APPLICATION,
+        status: FilingStatus.COMPLETED,
+        submittedDate: 'Sun, 02 Jun 2019 19:22:59 GMT'
+      } as any
+    ])
+
+    const wrapper = shallowMount(EntityHeader, {
+      vuetify,
+      router,
+      propsData: { businessId: 'BC1234567', tempRegNumber: null }
+    })
+    await Vue.nextTick()
+
+    expect(wrapper.find('#entity-legal-name').text()).toBe('Numbered Limited Company')
+    expect(wrapper.find('#business-description').text()).toBe('BC Limited Company')
+  })
+
+  it('displays draft numbered amalgamation application - Vertical', async () => {
```

Comment:
Again, there doesn't seem to be a difference between horizontal and vertical (except for the display name). What do you think about combining/reducing these tests, maybe with a comment saying they're the same for H and V?

Reply:
Same as above I will reduce these with a comment.